flowey/flowey_lib_common/src/run_cargo_nextest_run.rs

```diff
@@ -60,12 +61,13 @@ pub mod build_params {
         /// Whether to disable default features
         pub no_default_features: bool,
         /// Whether to build tests with unstable `-Zpanic-abort-tests` flag
-        // FIXME: this should probably be a generic `with_env` parameter
         pub unstable_panic_abort_tests: Option<PanicAbortTests>,
         /// Build unit tests for the specified target
         pub target: target_lexicon::Triple,
         /// Build unit tests with the specified cargo profile
         pub profile: CargoBuildProfile,
+        /// Additional env vars set when building the tests
+        pub extra_env: Option<BTreeMap<String, String>>,
```

Comment:
why have this wrapped in an option? Just pass an empty BTreeMap::new() if you don't need extra vars

Reply:
Sounds good, just wasn't sure what was most idiomatic here. Does BtreeMap::new() allocate anything if you never add to it?

Reply:
Not only does it not allocate - its also totally `const`

https://doc.rust-lang.org/std/collections/struct.BTreeMap.html#method.new